src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -463,8 +509,13 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         console.error(error);
         /* eslint-enable no-console */
 
-        // eslint-disable-next-line react/prop-types
-        this.props.onError(error, code, contextInfo);
+        const { onAnnotationsFetchError, onError } = this.props;
+
+        onError(error, code, contextInfo);
+
+        if (code === ERROR_CODE_FETCH_ANNOTATIONS) {
+            onAnnotationsFetchError({ ...error, origin: ORIGIN_ACTIVITY_SIDEBAR });
```

Comment:
It looks like this would lead to ContentSidebar/ContentPreview's `onError` prop being called twice for the same error. Can the Annotations HOC be limited to only managing the `annotator` instance?